lms/djangoapps/instructor_task/tasks_helper/utils.py

```diff
@@ -82,7 +82,9 @@ def csv_direct(self, writer):
         if self.should_msg_be_displayed:
             msg = self.process_message()
             encode = unicode(msg).encode('utf-8')
-            return writer.writerow([encode])
+            writer.writerow([encode])
+
+        return writer
```

Comment:
@jagonzalr 
I think it's quite OK if this method doesn't return anything. What's your opinion? ;)